flatdata-cpp/test/MultiVectorTest.cpp

```diff
@@ -251,6 +277,20 @@ TEST( MultiVectorTest, IterateOneTypeElementsRandomlyPlaced )
     ASSERT_TRUE( has_c2 );
 }
 
+TEST( MultiVectorTest, FlushingWhileBuilding )
+{
+    auto view = create_view_with_enough_items_to_flush( );
+
+    for ( size_t i = 0; i < NUM_ITEM_TO_FLUSH; i++ )
+    {
+        auto iter = view.second.iterator< CStruct >( i );
+        ASSERT_TRUE( iter.valid( ) ) << "Index " << i;
```

Comment:
Message: "Expected valid iterator at index: "

Reply:
Done automatically by GTest (it does expected + actual + user message). Made the message better, though.